modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -283,7 +283,13 @@ function synergywholesaledomains_getConfigArray(array $params)
         'defaultDnsConfig' => [
             'FriendlyName' => 'Default DNS Config',
             'Type' => 'dropdown',
-            'Options' => ['Nothing', 'Names Servers', 'FreeDNS with email forwarding', 'Parked', 'FreeDNS'],
+            'Options' => [
+                0 => 'Nothing',
```

Comment:
Since further down we're checking for string can we either changed these keys to strings, or check against integers in places like line 640?